resources/definitions/fdmprinter.def.json

```diff
@@ -2296,6 +2311,21 @@
                             "limit_to_extruder": "top_bottom_extruder_nr",
                             "settable_per_mesh": true
                         },
+                        "acceleration_ironing":
+                        {
+                            "label": "Ironing Acceleration",
+                            "description": "The acceleration with which ironing is performed.",
+                            "unit": "mm/s²",
+                            "type": "float",
+                            "minimum_value": "0.1",
+                            "minimum_value_warning": "100",
+                            "maximum_value_warning": "10000",
+                            "default_value": 3000,
+                            "value": "acceleration_topbottom",
+                            "enabled": "resolveOrValue('acceleration_enabled') and ironing_enabled",
```

Comment:
acceleration_enabled is not really a property which can be different between the extruders; it doesn't make sense to have a resolve function for it.

However, the value of the setting is already retrieved inconsistently, because sometimes it's just retrieved without calling resolveOrValue...
[not blocking]